packages/twenty-front/src/modules/page-layout/widgets/graph/graphWidgetBarChart/hooks/useBarChartTooltip.ts

```diff
@@ -8,42 +8,50 @@ import { type BarDatum, type ComputedDatum } from '@nivo/bar';
 import { isDefined } from 'twenty-shared/utils';
 
 type UseBarChartTooltipProps = {
-  hoveredBar: { key: string; indexValue: string | number } | null;
   enrichedKeys: BarChartEnrichedKey[];
   data: BarChartDataItem[];
   indexBy: string;
   formatOptions: GraphValueFormatOptions;
   enableGroupTooltip?: boolean;
+  layout?: 'vertical' | 'horizontal';
 };
 
 export const useBarChartTooltip = ({
```

Comment:
Can we create a util instead of a hook here ? Because there isn't any reactive dependency.